samples/testing-frameworks/appium/client-side/java/src/test/resources/desiredCapabilities.android.clientside.properties

```diff
@@ -1,8 +1,9 @@
 platformName=Android
-deviceName=Android Device
 fullReset=false
 noReset=true
+automationName=uiautomator2
 bitbar_target=Android
```

Comment:
`bitbar_target` is deprecated and should be removed

Reply:
bump

Reply:
Done